### PR TITLE
[Spice internet-indicator] creation of a new applet :)

### PR DESCRIPTION
**Pull request by sangorys**
**Dear cinnamon team** : I would like to publish my applet into Cinnamon Spices. I guess that some people would love it. It is a very simple applet which displays if Internet works or does not work by displaying a simple icon. In the background a simple "ping www.google.fr -c 1 -w 1000" is run every 3 or 10 seconds according to the configuration. Initially : I often work in a train with a connection with my phone. I never know if my laptop has Internet or lost Internet ! This is the reason why I created the applet for myself. But maybe the open-source community could be also interested by it ? I did not find any equivalent in Cinnamon Spices. 

`I am a beginner with git, javascript, cinnamon applet and gitHub. Please help me if I am doing anything wrong. I am ready to learn and to improve anything. Please contact me for anything. It is the first time I colaborate to an open source project !!!`

Thank you for your collaboration

### CODE REVIEW

File: internet-indicator@sangorys/files/internet-indicator@sangorys/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Cinnamenu
```
Comment (Gr3q) [inline internet-indicator@sangorys/files/internet-indicator@sangorys/README.md:1]: This is Cinnamenu's README

File: internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+		//global.logError(stderr.toString().length);
+		
+		if (stderr == "")
+		{
+			//global.logError("ping OK");  
+			if (stderr.toString() != oldStderr)
+			{
+			  //global.logError("connecté");
+			  oldStderr = stderr.toString();
+			  Util.spawnCommandLine("/usr/sbin/notify-send 'Internet OK'");
```
Comment (Gr3q) [inline internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js:228]: You are sending an untranslated string to notify

File: internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+			//else {global.logError("Pas de changement : connecté");}
+		}
+		else
+		{
+			//global.logError("ping failed");  
+			if (stderr.toString() != oldStderr)
+			{
+			  //global.logError("déconnecté");
+			  oldStderr = stderr.toString();
+			  Util.spawnCommandLine("/usr/sbin/notify-send 'Pas d internet'");
```
Comment (Gr3q) [inline internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js:242]: You are sending an untranslated string to notify in French

**Review by Gr3q — COMMENTED** [review]
Apart from some things, I think it's worthwhile for you to:

* Clean the commented and unused code out so it's easier to review
* and use svg icons (symbolic) instead so it integrates with themes better, but it's just a preference of mine.

File: internet-indicator@sangorys/files/internet-indicator@sangorys/README.md (hunk 1 above)
Comment (sangorys) [reply internet-indicator@sangorys/files/internet-indicator@sangorys/README.md:1]: First of all : thanks for all your valuable comments !!!

My bad. This file should even not exist in this folder. The correct README.md is the one in the root internet-indicator@sangorys folder.
I removed the wrong README.md

File: internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js (hunk 2 above)
Comment (sangorys) [reply internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js:228]: Yes, this is something I want to improve but I don't know yet how to use "language dictionnary" to have my applets translated for each language.
Can you help me with an example, a tuto or a simple explanation ?

File: internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js (hunk 3 above)
Comment (sangorys) [reply internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js]: My bad. I fixed it

**Comment by sangorys** [comment]
> Apart from some things, I think it's worthwhile for you to:
> 
>     * Clean the commented and unused code out so it's easier to review
> 
>     * and use svg icons (symbolic) instead so it integrates with themes better, but it's just a preference of mine.

You are right. Usufull comments...

I removed some comments and unused code even if I have not finished yet. I let some of them because I will use them in the futur.

Agree for svg icons. In the same time, I don't have the svg skills yet.

Thanks  Gr3q for your comments, it is important for me to learn the best practices as soon as possible  :)

File: internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js (hunk 2 above)
Comment (Gr3q) [reply internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js]: As far as I know there is no pre-injected function named `_()` so you need to write your own. There is a chance I'm wrong tho, so do this last.

You did generate the translations correctly, but you will need to install it as well to test it.
You can do that with `cinnamon-spices-makepot "internet-indicator@sangorys" --install`

Comment (sangorys) [reply internet-indicator@sangorys/files/internet-indicator@sangorys/applet.js]: You are great support ! Thanks.

Translation now works.

I guess everything works now for a 1 version. No ?

**Comment by sangorys** [comment]
Hello,

I fixed all comments from Gr3q (thanks a lot for him !!!), what are the next steps now ?

**Comment by Gr3q** [comment]
You wait for @brownsr

**Comment by sangorys** [comment]
> You wait for @brownsr

Usually, do you know how much time it could take ?

**Comment by Gr3q** [comment]
Usually not this long, but it's a new applet

**Pull request merged**